in-work/linear_equations.md

```diff
@@ -27,25 +29,20 @@ In this lecture we discuss linear equations and their solutions.
 
 We also discuss how to compute the solutions with matrix algebra.
 
-Finally, we introduce eigenvalues and show how they can be useful for economic
-problems.
-
 We assume that students are familiar with matrices and understand the basics
 of matrix algebra.
 
 We will use the following imports:
 
-
```

Comment:
@maanasee these will need to use `code-cell` directives for them to execute. would you mind changing these `syntax highlight` code snippets to use `code-cell`?

Reply:
Thanks @mmcky! I have made the changes I hope this is what was required.